node/.env.example

```diff
@@ -52,3 +52,18 @@ POR_PROVIDER_URL=
 # (optional) defaults to 3000
 POR_PORT=
 
+# DAL
+
+# (optoinal) defaults to 8090
+DAL_API_PORT=
+
+# (required)
+# KAIA_WEBSOCKET_URL=
+
+# (required)
+# SUBMISSION_PROXY_CONTRACT=
+
+# (required)
+# DATABASE_URL=
+# REDIS_HOST=
+# REDIS_PORT=
```

Comment:
a minor suggestion, how about uncommenting required variables?

Reply:
thanks for the suggestion, I commented this because the same variables are already declared above, do you think it'd be okay to have duplicate declaration inside .env.example file? if so, I'll uncomment these 🙏

Reply:
if they are already declared above i think no need, i didn't notice the ones at the top